locales/en/translation.json

```diff
@@ -433,7 +433,7 @@
   "ai:Number of hosts": "Number of hosts",
   "ai:OCS requirements": "OCS requirements",
   "ai:ODF requirements": "ODF requirements",
-  "ai:Only {{hostsSelected}} hosts are selected.": "Only {{hostsSelected}} hosts are selected.",
+  "ai:Only {{hostsSelected}} hosts are selected.": "Only {{hostsSelected}} host is selected.",
```

Comment:
Shouldn't the text in the code reflect the singular case? To keep the same pattern everywhere
What I mean, is that the key would be "Only {{hostsSelected}} host is selected."

Reply:
Yes, can be a good option. Changed!